std/protocols/lookup.asm

```diff
@@ -112,6 +112,15 @@ let lookup: Constr, expr -> () = constr |lookup_constraint, multiplicities| {
     is_first * acc_2 = 0;
     constrain_eq_ext(update_expr, from_base(0));
 
+    // Build an annotation for witness generation
+    let witgen_annotation = match lookup_constraint {
```

Comment:
Can you add a function to `std::constraints` like `to_phantom(lookup_constraint, multiplicities)`?